core/rid_owner.h

```diff
@@ -298,7 +298,11 @@ class RID_Alloc : public RID_AllocBase {
 			if (description) {
 				print_error("ERROR: " + itos(alloc_count) + " RID allocations of type '" + description + "' were leaked at exit.");
 			} else {
+#if defined(_CPPRTTI)
```

Comment:
I think we'd usually use `NO_SAFE_CAST` (defined for Android and Web) for this (here `#if !defined(NO_SAFE_CAST)`, or changing the order).